bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/widgets/Widget.java

```diff
@@ -2647,6 +2647,11 @@ void notifyDisposalTracker() {
 	}
 }
 
+GC getNewGC(long hDC, GCData data) {
```

Comment:
This is not a getter but a creator for a new GC. So it should be rather called `createNewGC` or the like.

Reply:
That will be changed/removed after our discussion yesterday anyway

Reply:
So I'll wait for the finalizing of what needs to be added and then directly add those changes (directly using the monitor zoom).
@HeikoKlare for your context, this method is in the parent of all the consumers who were creating the GC and were supposed to pass their native zoom in ther GCData. For now they just call this method which is defined in their parents which inserts the nativeZoom more like an interceptor and then creates a new GC with it. But we'll change it later.